src/main/java/core/basesyntax/strategy/impl/ReturnOperationImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class ReturnOperationImpl implements OperationHandler {
+    @Override
+    public void applyOperation(FruitTransaction transaction) {
+        if (transaction.getQuantity() <= MIN_QUANTITY) {
+            throw new RuntimeException("You must return 1 or more items. But was: "
+                    + transaction.getQuantity());
```

Comment:
The constant 'MIN_QUANTITY' is not defined within this class or imported from elsewhere. You should define this constant in the class to avoid compilation errors.